public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
Please remove files you are not using e.g `public/logo512.png, public/logo192.png and public/favicon.ico`

Reply:
Done.
Replaced unused files with the proper files of our own